aiida_common_workflows/workflows/relax/fleur/workchain.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+"""
+Implementation of `aiida_common_workflows.common.relax.workchain.CommonRelaxWorkChain`
+for FLEUR (www.flapw.de).
+"""
+from aiida import orm
+from aiida.engine import calcfunction
+from aiida.plugins import WorkflowFactory
+
+from ..workchain import CommonRelaxWorkChain
+
+__all__ = ('FleurCRelaxWorkChain',)
+
+AiidaFleurRelaxWorkChain = WorkflowFactory('fleur.base_relax')
+
+
+@calcfunction
+def get_forces_from_trajectory(trajectory):  # pylint: disable=unused-argument
+    """Calcfunction to get forces from trajectory"""
+    forces = orm.ArrayData()
+    # currently the fleur relax workchain does not output trajectory data,
+    # but it will be adapted to do so
+    # largest forces are found in workchain output nodes
+    # forces.set_array(name='forces', array=trajectory.get_array('forces')[-1])
+    return forces
+
+
+@calcfunction
+def get_total_energy(parameters):
+    """Calcfunction to get total energy from relax output"""
+    return orm.Float(parameters.get_attribute('energy'))
+
+
+class FleurCRelaxWorkChain(CommonRelaxWorkChain):
+    """
+    Implementation of `aiida_common_workflows.common.relax.workchain.CommonRelaxWorkChain`
+    for FLEUR.
+    """
+
+    _process_class = AiidaFleurRelaxWorkChain
+
+    def convert_outputs(self):
+        """Convert the outputs of the sub workchain to the common output specification."""
+        self.out('relaxed_structure', self.ctx.workchain.outputs.optimized_structure)
+        self.out('total_energy', get_total_energy(self.ctx.workchain.outputs.out))
+        self.out('forces', get_forces_from_trajectory(self.ctx.workchain.outputs.out))
```

Comment:
Is the output containing the energy in forces in the fleur plugin really called `out`?

Reply:
> Is the output containing the energy in forces in the fleur plugin really called `out`?

Currently yes, choice by Vasily. I also do not like it, but for now I am reluctant to change it.
I would also like to merge the 'base_relax' and the relax workchain of Fleur.

Reply:
No need to change anything. just making sure that this was not an accidental typo. Did you get a chance to test this on QM? Or at least locally? If so, then I will approve and merge this

Reply:
Yes I ran it locally and on QM (release from March) weeks ago, that's how I knew the example was not in the package. I could test it tomorrow on the new QM release from last week.